### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by ehannes**
It was time to update the readme :)

Here's how the new configuration table in the readme looks like:
Screenshot 2023-01-18 at 10 44 21

### CODE REVIEW

File: DEVELOPER_README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Readme for Developers
```
Comment (ehannes) [inline DEVELOPER_README.md:1]: Extracted from `README.md`

File: CONTRIBUTE.md [hunk 2]
```diff
@@ -1,24 +1,35 @@
...
+
+To contribute:
+
+1. Fork it ( http://github.com/accodeing/fortnox-api/fork )
+2. Clone your fork
+   (`git clone https://github.com/<your GitHub user name>/fortnox-api.git`)
 3. Create your feature branch (`git checkout -b my-new-feature`)
 4. Commit your changes (`git commit -am 'Add some feature'`)
 5. Push to the branch (`git push origin my-new-feature`)
 6. Create new Pull Request
```
Comment (d-Pixie) [inline CONTRIBUTE.md:17]: " against our develop branch"

File: CONTRIBUTE.md [hunk 3]
```diff
@@ -1,24 +1,35 @@
...
-https://github.com/my-codeworks/fortnox-api` ). Then you can do something like
-this to update your fork:
+remote server to your clone (
+`git remote add upstream https://github.com/accodeing/fortnox-api` ). Then you
+can do something like this to update your fork:
 
 1. Fetch branches from upstream (`git fetch upstream`)
 2. Checkout your master branch (`git checkout master`)
 3. Update it (`git rebase upstream/master`)
 4. And push it to your fork (`git push origin master`)
```
Comment (d-Pixie) [inline CONTRIBUTE.md:29]: Might need `--force` here depending on ....

File: DEVELOPER_README.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+## Rubocop
+
+When updating Rubocop in `fortnox-api.gemspec`, you need to set the explicit
+version that codeclimate runs in `.codeclimate.yml`
+
+## Updating Ruby version
+
+When updating the required Ruby version, you need to do the following:
+
+- Bump Ruby version in `fortnox-api.gemspec`
```
Comment (d-Pixie) [inline DEVELOPER_README.md:39]: .tool-versions? Or are we not using asdf for this gem yet?

**Review by d-Pixie — CHANGES_REQUESTED** [review]

File: DEVELOPER_README.md (hunk 4 above)
Comment (ehannes) [reply DEVELOPER_README.md:39]: Good catch!

**Pull request merged**